R/database.R

```diff
@@ -1346,22 +1396,24 @@ get_ashm_count_matrix = function(regions_bed,
                                  maf_data,
                                  sample_metadata,
                                  use_name_column = FALSE,
-                                 from_indexed_flatfile = FALSE,
-                                 allow_clustered = FALSE){
+                                 from_indexed_flatfile = TRUE){
 
   if(missing(regions_bed)){
     regions_bed = grch37_ashm_regions
   }
   ashm_maf = get_ssm_by_regions(regions_bed = regions_bed,
-                                streamlined = TRUE,
+                                basic_columns = TRUE,
                                 maf_data = maf_data,
                                 use_name_column = use_name_column,
-                                from_indexed_flatfile = from_indexed_flatfile,
-                                allow_clustered = allow_clustered)
+                                from_indexed_flatfile = from_indexed_flatfile)
 
   ashm_counted = ashm_maf %>%
-    group_by(sample_id, region_name) %>%
+    group_by(Tumor_Sample_Barcode, Region_Name) %>%
     tally()
+
+  colnames(ashm_counted)[1] = "sample_id"
+  colnames(ashm_counted)[2] = "region_name"
```

Comment:
The `get_ssm_by_regions` below specifies this to be `colnames(r_names)[1] <- "Region_Name"`. So basically in the internal function we are assigning one name, and here re-naming it. Will it make sense to assign names in one place and then use it consistently? What is the default name, maybe we do not need to re-name it at all? I'll leave a comment at the line that I mean